nodejs/week2/warmup/numbers-router.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/add", async(request, response) => {
+    const firstNumber = parseInt(request.query.firstNumber);
+    const secondNumber = parseInt(request.query.secondNumber);
+    const add = firstNumber + secondNumber;
+    if (firstNumber && secondNumber) {
+        response.send({ data: [add] });
+    }
+});
+router.get("/multiply/:first/:second", async(request, response) => {
+    const first = Number(request.params.first);
+    const second = Number(request.params.second);
+    const multiply = first * second;
```

Comment:
What would happen if either of first or second would not be a number?

Reply:
I added an if, else block, to check if each parameter is not a number. And if it is not - response with error.